packages/core/src/events/DefaultEventHandlers.ts

```diff
@@ -148,6 +148,9 @@ export class DefaultEventHandlers<O = {}> extends CoreEventHandlers<
               return;
             }
 
+            const ancestors = store.query.node(targetId).ancestors();
+            store.actions.setNodeEvent('draggedOver', [targetId, ...ancestors]);
+
```

Comment:
I'm wondering if we really need to include the ancestors here. Could we get away with just the `targetId` being the Node that's being dragged over? 

Then in the use case where we need to know if the current ancestor element contains the Node that's being dragged over, we could just do:

```tsx
// Inside a parent Node
const descendants = query.node(id).descendants();
// Not a nice api, but maybe a helper could be used:
const draggedOverDescendant = Array.from(state.events.draggedOver).find(id => descendants.includes(id))
```

Reply:
I prefer having access to all the node ids that we drag over. But it's not a strong opinion, so I don't mind changing the API.